packages/terra-form-select/src/terra-dev-site/doc/form-select/SelectField.2.doc.jsx

```diff
@@ -26,6 +28,11 @@ const DocPage = () => (
       title: 'Select Field Example',
       example: <SelectFieldExample />,
       source: SelectFieldExampleSrc,
+    }, {
+      title: 'Maximum Selection',
+      description: 'A multiple variant that allows up to 3 selections',
```

Comment:
```suggestion
      description: 'A multiple selection variant that allows up to a set number of selections. (Not applicable to  \'default\', \'combobox\', and \'search\' variants) ',
```


(applies to only)',
      description: 'An example with an option to clear the selected item. Not applicable to \'multiple\' or \'tag\' variants.',

Reply:
Updated the example description: https://github.com/cerner/terra-core/pull/2244/commits/3c8375b9ae7e72f9eb55a40a15eab66819461ff7